src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,39 @@
+import { Link } from "react-router-dom";
+// import { useProducts } from "../../context/product-context";
+import "./navbar.css";
+
+export function Navbar(){
+    // const {state} = useProducts();
```

Comment:
Can remove un-used code if not required.

Reply:
Thank you for pointing it out